libraries/Microsoft.Bot.Connector/Teams/TeamsOperations.cs

```diff
@@ -139,24 +128,14 @@ public async Task<HttpOperationResponse<TeamDetails>> FetchTeamDetailsWithHttpMe
                 throw new ValidationException(ValidationRules.CannotBeNull, "teamId");
             }
 
-            // Tracing
-            bool shouldTrace = ServiceClientTracing.IsEnabled;
-            string invocationId = null;
-            if (shouldTrace)
-            {
-                invocationId = ServiceClientTracing.NextInvocationId.ToString(CultureInfo.InvariantCulture);
-                Dictionary<string, object> tracingParameters = new Dictionary<string, object>();
-                tracingParameters.Add("teamId", teamId);
-                tracingParameters.Add("cancellationToken", cancellationToken);
-                ServiceClientTracing.Enter(invocationId, this, "FetchTeamDetails", tracingParameters);
-            }
+            var invocationId = TraceActivity("FetchTeamDetails", new { teamId }, cancellationToken);
 
             // Construct URL
             var baseUrl = Client.BaseUri.AbsoluteUri;
```

Comment:
I think this is not beign used anymore? right?

Reply:
what is not needed?